assets/js/omniSelectEngine.js

```diff
@@ -0,0 +1,394 @@
+$(document).ready(function ($) {
```

Comment:
Plase reformatte to have an empty line between declarations `});\n`

Reply:
Done